README.md

```diff
@@ -1,19 +1,19 @@
 # Uptane Standard
 
-[Uptane](https://uptane.github.io) is the first compromise-resilient software update security system for the automotive industry. Beginning in 2018, a working group has begun the process of describing the system's design, implementation, and deployment considerations as a formal standard, working as an [IEEE-ISTO](https://ieee-isto.org/) industry group and is currently a [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/) project. This repository is the public home of that standardization work.
+[Uptane](https://uptane.github.io) is the first compromise-resilient software update security system for the automotive industry. In 2018, a working group began the process of describing the system's design, implementation, and deployment considerations as a formal standard. Working as an [IEEE-ISTO](https://ieee-isto.org/) industry group, Volume 1.0.0 of *Uptane Standard for Design and Implementation* was released in July 2019. The Uptane standards initiative is now under the umbrella of the [Linux Foundation Joint Development Foundation](http://www.jointdevelopment.org/). This repository is the public home of that standardization work.
 
 ## Existing documentation
 
-Several documents are already available which describe Uptane's design and implementation. Note that an intial version of the Uptane Standards document has been developed and should be considered the authoritative resource for the framework:
+Several documents are already available which describe Uptane's design and implementation. Note that *Uptane Standard for Design and Implementation* should be considered the authoritative resource for the framework:
 
-* [Uptane Standards Document](https://uptane.github.io/uptane-standard/uptane-standard.html)
+* [Uptane Standards Document V. 1.0.0](https://uptane.github.io/uptane-standard/uptane-standard.html)
```

Comment:
Should this also refer to the IEEE/ISTO version number?

Reply:
Yes, like I said [here](https://github.com/uptane/uptane.github.io/pull/16#issuecomment-521359913), the IEEE-ISTO links should point to a static copy of the document instead of the current, ever-evolving one.

@patrickvacek Could you help to quickly build and publish the 1.0.0 IEEE-ISTO standard?

Reply:
Isn't that already [here](https://github.com/uptane/uptane-standard/releases)?

Reply:
I believe what is wanted, Patrick, is a static PDF version of V. 1.0.0 i.e. a PDF

Reply:
Have we ever compiled a PDF version of the standard? Unfortunately I don't have a handy tool to do that. @tkfu may have some ideas, but I'm unfortunately not likely to be much help with that.

Reply:
Would the document at https://uptane.github.io/uptane-standard/1.0.0/uptane-standard.html work?